cores/rp2040/wiring_analog.cpp

```diff
@@ -121,6 +124,14 @@ extern "C" void analogWrite(pin_size_t pin, int val) {
 
 auto_init_mutex(_adcMutex);
 static int _readBits = 10;
+static int _lastADCMux = 0;
+static bool _adcGPIOInit[29];
```

Comment:
Optimization nitpick: A `uint32_t` used as a bitmask could have save 25 bytes of memory.

Reply:
Optimizing the optimization?  That's a recipe for an infinite loop or black hole! :laughing: